openshift/recovery-controller/src/main/java/me/snowdrop/boot/narayana/openshift/recovery/NarayanaRecoveryTerminationControllerAutoConfiguration.java

```diff
@@ -52,4 +52,9 @@ public LogScrapingRecoveryErrorDetector logScrapingRecoveryErrorDetector(Statefu
         return new LogScrapingRecoveryErrorDetector(properties.getCurrentPodName(), properties.getLogScrapingErrorDetectionPattern());
     }
 
+    @Bean
```

Comment:
This adds the correct error detector, but the `LogScrapingRecoveryErrorDetector` above should be changed to `matchIfMissing = false` to completely disable it by default.

Reply:
Thanks for pointing it out ! I will correct it.